src/js/common/polyfills.js

```diff
@@ -24,3 +24,13 @@ if (!Modernizr.fetch) {
 
 // This polyfill has its own test logic so no need to conditionally require.
 require('polyfill-function-prototype-bind');
+
+// Disable scrolling on input[type="number"] because it was changing the value unexpectedly
+// NOTE: This also just stops the user from scrolling when hovering over a number
+//  input that has focus. This isn't ideal, but until a solution is found to that,
+//  this is what we've got.
+document.addEventListener('mousewheel', (event) => {
```

Comment:
This can be pulled into a different file if it makes more sense to do that. Also, I can make this more like:

```js
export function preventScrolling(element) {
  element.addEventListener('mousewheel', (event) => {
    if (event.target.type === 'number' && document.activeElement === event.target) {
      event.preventDefault();
    }
  });
}
```
So we can call it only when we want to instead of making this global on `document`.

I'm not super fond of this, but it works. Mostly.